modules/imgproc/test/test_houghlines.cpp

```diff
@@ -196,6 +219,33 @@ void BaseHoughLineTest::run_test(int type)
 #endif
 }
 
+void HoughLinesPointSetTest::run_test(void)
+{
+    Mat lines;
+    vector<Point2f> point;
```

Comment:
Please add also a check for integer point coordinates (the values can be the same)